src/main/java/com/nashtech/rookie/yasa/controller/GlobalExceptionHandler.java

```diff
@@ -17,12 +19,41 @@ public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
                 ProductNotFoundException.class,
                 RatingNotFoundException.class,
                 UserNotFoundException.class,
-                CartNotFoundException.class
```

Comment:
Why do you put this class in controller package? It should be placed in exception package